src/Library/demos/Toggle Button/main.blp

```diff
@@ -0,0 +1,89 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Toggle Button";
+  description: _("Represent active-state visually");
+
+  Box {
+    orientation: vertical;
+
+    Box Grouped_Toggle_Buttons {
+      orientation: horizontal;
+      halign: center;
+      margin-bottom: 12;
+      styles ["linked"]
+
+      ToggleButton button_first {
+        active: true;
+        icon-name: "eye-not-looking-symbolic";
+      }
+
+      ToggleButton button_second {
+        active: false;
+        icon-name: "eye-open-negative-filled-symbolic";
+        group: button_first;
+      }
+    }
+
+    Label {
+      label: "Grouped Toggle Buttons";
+      justify: center;
+      margin-bottom: 18;
+    }
+
+    Box Ungrouped_Toggle_Buttons {
+      orientation: horizontal;
+      halign: center;
+      margin-bottom: 12;
+
+      ToggleButton button_third {
+        active: false;
+        icon-name: "photo-camera-symbolic";
+        halign: center;
+        margin-end: 6;
+      }
+
+      ToggleButton button_fourth {
+        active: false;
+        icon-name: "flashlight-symbolic";
+        halign: center;
+      }
+    }
+
+    Label {
+      label: "Ungrouped Toggle Buttons";
+      margin-bottom: 18;
+      justify: center;
+    }
+
+    ToggleButton console {
```

Comment:
let's be consistent and use the same format for all ids, please follow this scheme:

```suggestion
    ToggleButton button_console {
```